setup.py

```diff
@@ -51,7 +51,7 @@ def read(filename):
 
     packages=PEP420PackageFinder.find(include=["lumicks.*"]),
     python_requires='>=3.6',
-    install_requires=['pytest>=3.5', 'h5py>=2.9, <3.0', 'numpy>=1.14, <2',
+    install_requires=['pytest>=3.5', 'h5py>=3.0, <4', 'numpy>=1.14, <2',
```

Comment:
This might be a good time to re-format this list to one entry per line in order to get nicer diffs.

Reply:
Yes. Done.